vips/vip-192/commands-testnet.ts

```diff
@@ -0,0 +1,345 @@
+    params: [PRIME, "togglePause()", NORMAL_TIMELOCK],
+  },
+  {
+    target: ACM,
+    signature: "giveCallPermission(address,string,address)",
+    params: [PRIME, "updateAlpha(uint128,uint128)", FAST_TRACK_TIMELOCK],
+  },
+  {
+    target: ACM,
+    signature: "giveCallPermission(address,string,address)",
+    params: [PRIME, "updateMultipliers(address,uint256,uint256)", FAST_TRACK_TIMELOCK],
+  },
+  {
+    target: ACM,
+    signature: "giveCallPermission(address,string,address)",
+    params: [PRIME, "setStakedAt(address[],uint256[])", FAST_TRACK_TIMELOCK],
+  },
+  {
+    target: ACM,
+    signature: "giveCallPermission(address,string,address)",
+    params: [PRIME, "addMarket(address,uint256,uint256)", FAST_TRACK_TIMELOCK],
+  },
+  {
+    target: ACM,
+    signature: "giveCallPermission(address,string,address)",
+    params: [PRIME, "setLimit(uint256,uint256)", FAST_TRACK_TIMELOCK],
+  },
+  {
+    target: ACM,
+    signature: "giveCallPermission(address,string,address)",
+    params: [PRIME, "setMaxLoopsLimit(uint256)", FAST_TRACK_TIMELOCK],
+  },
+  {
+    target: ACM,
+    signature: "giveCallPermission(address,string,address)",
+    params: [PRIME, "issue(bool,address[])", FAST_TRACK_TIMELOCK],
+  },
+  {
+    target: ACM,
+    signature: "giveCallPermission(address,string,address)",
+    params: [PRIME, "burn(address)", FAST_TRACK_TIMELOCK],
+  },
+  {
+    target: ACM,
+    signature: "giveCallPermission(address,string,address)",
+    params: [PRIME, "togglePause()", FAST_TRACK_TIMELOCK],
+  },
+  {
+    target: ACM,
+    signature: "giveCallPermission(address,string,address)",
+    params: [PRIME, "updateAlpha(uint128,uint128)", CRITICAL_TIMELOCK],
+  },
+  {
+    target: ACM,
+    signature: "giveCallPermission(address,string,address)",
+    params: [PRIME, "updateMultipliers(address,uint256,uint256)", CRITICAL_TIMELOCK],
+  },
+  {
+    target: ACM,
+    signature: "giveCallPermission(address,string,address)",
+    params: [PRIME, "setStakedAt(address[],uint256[])", CRITICAL_TIMELOCK],
+  },
+  {
+    target: ACM,
+    signature: "giveCallPermission(address,string,address)",
+    params: [PRIME, "addMarket(address,uint256,uint256)", CRITICAL_TIMELOCK],
+  },
+  {
+    target: ACM,
+    signature: "giveCallPermission(address,string,address)",
+    params: [PRIME, "setLimit(uint256,uint256)", CRITICAL_TIMELOCK],
+  },
+  {
+    target: ACM,
+    signature: "giveCallPermission(address,string,address)",
+    params: [PRIME, "setMaxLoopsLimit(uint256)", CRITICAL_TIMELOCK],
+  },
+  {
+    target: ACM,
+    signature: "giveCallPermission(address,string,address)",
+    params: [PRIME, "issue(bool,address[])", CRITICAL_TIMELOCK],
+  },
+  {
+    target: ACM,
+    signature: "giveCallPermission(address,string,address)",
+    params: [PRIME, "burn(address)", CRITICAL_TIMELOCK],
+  },
+  {
+    target: ACM,
+    signature: "giveCallPermission(address,string,address)",
+    params: [PRIME, "togglePause()", CRITICAL_TIMELOCK],
```

Comment:
`GUARDIAN` should be granted to execute `togglePause()` also on Prime (also on mainnet)